### ISSUE TITLE
Column mode

### ISSUE THREAD

**Issue by dunpealer** [akahuku/wasavi#104]
Wasavi is nearly perfect, except for column mode support, which is very useful when formatting text or code. Are there any plans to support it?

**Comment by akahuku** [comment]
Is the "column mode" block-visual mode in vim?
or virtual edit state in vim?

**Comment by sirex** [comment]
I think he is talking about [blockwise-visual](http://vimdoc.sourceforge.net/htmldoc/visual.html#CTRL-V).

I would like to see it two, really often used thing, especially when formatting emails.

**Comment by dunpealer** [comment]
@sirex yes, that's exactly what I'm talking about. Super useful feature, and one of the biggest reasons I have to jump into GVim for quick editing sessions on textareas.

**Comment by kitsu** [comment]
+15
https://www.bountysource.com/issues/27872283-column-mode

**Comment by martinvonwittich** [comment]
I was hoping wasavi would have this feature when I installed it today, so I hope this gets implemented soon :)
I added 15$ to the bounty.

**Comment by turkus** [comment]
Up. Would it be implemented?
